nib.gemspec

```diff
@@ -24,13 +24,9 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency('gli', '~> 2.16')
 
-  s.add_development_dependency('codeclimate-test-reporter', '~> 1.0.7')
-  s.add_development_dependency('did_you_mean')
-  s.add_development_dependency('guard')
-  s.add_development_dependency('guard-rspec')
-  s.add_development_dependency('guard-rubocop')
```

Comment:
The primary reason for these removals is that Guard has a dependency on `ffi` which has a native extension and fails to build on stock `-alpine`/`-slim` image variants. Rather than adding the overhead of installing additional dependencies multiple times in the CI pipeline, I chose to remove Guard altogether. We can revisit in the future if this becomes a concern.